requirements-test.txt

```diff
@@ -88,10 +88,6 @@ coverage[toml]==5.5 \
     --hash=sha256:f0b278ce10936db1a37e6954e15a3730bea96a0997c26d7fee88e6c396c2086d \
     --hash=sha256:f11642dddbb0253cc8853254301b51390ba0081750a8ac03f20ea8103f0c56b6
     # via pytest-cov
-exceptiongroup==1.0.4 \
```

Comment:
pip-compiling this for python-3.11 causes the unit tests to fail on 3.9, 3.10 :frowning_face:

Reply:
It sounds like we need one requirements file per python version (or switch to a different package manager) :disappointed: 

https://github.com/jazzband/pip-tools/issues/639

Reply:
Oh, true. Come to think of it, I don't even know which python version dependabot uses :thinking:

Reply:
If we pip-compile for py39, do the py311 tests work?

It still wouldn't be an ideal solution, but probably good enough

Reply:
Yeah, it looks like that will work :+1: